pallets/capacity/src/lib.rs

```diff
@@ -462,6 +480,49 @@ pub mod pallet {
 			Self::deposit_event(Event::EpochLengthUpdated { blocks: length });
 			Ok(())
 		}
+
+		/// Sets the target of the staking capacity to a new target.
+		/// This adds a chunk to `StakingAccountDetails.stake_change_unlocking chunks`, up to `T::MaxUnlockingChunks`.
+		/// The staked amount and Capacity generated by `amount` originally targeted to the `from` MSA Id is reassigned to the `to` MSA Id.
+		/// Does not affect unstaking process or additional stake amounts.
+		/// ### Errors
+		/// - [`Error::StakingAccountNotFound`] if origin does not have a staking account
+		/// - [`Error::MaxUnlockingChunksExceeded`] if `stake_change_unlocking_chunks` == `T::MaxUnlockingChunks`
+		/// - [`Error::StakerTargetRelationshipNotFound`] if `from` is not a target for Origin's staking account.
+		/// - [`Error::StakingAmountBelowMinimum`] if `amount` to retarget is below the minimum staking amount.
+		/// - [`Error::InsufficientStakingBalance`] if `amount` to retarget exceeds what the staker has targeted to `from` MSA Id.
+		/// - [`Error::InvalidTarget`] if `to` does not belong to a registered Provider.
+		#[pallet::call_index(4)]
+		#[pallet::weight(T::WeightInfo::unstake())]
+		pub fn change_staking_target(
```

Comment:
are we really changing the staking target? It seems more like allotting (distributing, rationing) stake to another target.

Reply:
that's true, do you have another suggestion for the name?